render/wlr_renderer.c

```diff
@@ -251,23 +251,28 @@ bool wlr_renderer_init_wl_display(struct wlr_renderer *r,
 }
 
 struct wlr_renderer *wlr_renderer_autocreate_with_drm_fd(int drm_fd) {
+	struct wlr_renderer *renderer = NULL;
+	struct wlr_egl *egl = wlr_egl_create_from_drm_fd(drm_fd);
+	if (egl != NULL) {
+		goto create_renderer;
+	}
+
+	// Fallback to GBM
```

Comment:
I'd prefer the fallback to be in `wlr_egl_create_from_drm_fd` if possible. This would allow to re-use the `wlr_egl` returned by `egl_create` and avoid the double initialization and double log messages.

Reply:
Good idea, fixed in v2